src/site/content/en/secure/securely-hosting-user-data/index.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title: Securely hosting user data
+date: 2023-06-01
+description: >
+  How to search for vulnerabilities on your site.
+tags:
+  - security
+---
+
+Many web applications need to display user-controlled content. This can be as simple as serving user-uploaded images (for example, profile photos), or as complex as rendering user-controlled HTML (for example, a web development tutorial). This has always been difficult to do securely, so we've worked to find easy, but secure solutions that can be applied to most types of web applications.
+
+## Classical Solutions for Isolating Untrusted Content
+
+The classic solution for securely serving user-controlled content is to use what are known as ["sandbox domains''](https://security.googleblog.com/2012/08/content-hosting-for-modern-web.html). The basic idea is that if your application's main domain is `example.com`, you could serve all untrusted content on `exampleusercontent.com`. Since these two domains are [cross-site](/same-site-same-origin), any malicious content on `exampleusercontent.com` can't impact `example.com`.  
```

Comment:
```suggestion
The classic solution for securely serving user-controlled content is to use what are known as [sandbox domains](https://security.googleblog.com/2012/08/content-hosting-for-modern-web.html). The basic idea is that if your application's main domain is `example.com`, you could serve all untrusted content on `exampleusercontent.com`. Since these two domains are [cross-site](/same-site-same-origin), any malicious content on `exampleusercontent.com` can't impact `example.com`.  
```